blueocean-dashboard/src/main/js/redux/actions.js

```diff
@@ -212,6 +209,50 @@ exports.fetchJson = function fetchJson(url, onSuccess, onError) {
 };
 
 /**
+ * Fetch TXT/log data and repeat until the server tells us that there is no more
+ * <p>
+ * Utility function that can be mocked for testing.
+ *
+ * @param url The URL to fetch from.
+ * @param start query parameter tells API to send log starting from this offset in the log file.
+ * @param onSuccess Main callback to run specific callback code
+ * @param onError Error callback
+ */
+
+exports.fetchLogsReFetch = function fetchJson(url, start, onSuccess, onError) {
+    let refetchUrl;
+    if (start === null) {
+        refetchUrl = url;
+    } else {
+        refetchUrl = `${url}?start=${start}`;
+    }
+    fetch(refetchUrl, fetchOptions)
+        .then(checkStatus)
+        .then(response => {
+            /*
+             * If X-More-Data is true, then client should repeat the request after some delay.
+             * In the repeated request it should use X-TEXT-SIZE header value with start query parameter.
+             */
+            if (response.headers.get('X-More-Data')) {
+                /*
+                 * X-TEXT-SIZE is the byte offset of the raw log file client should use in the next request
+                 * as value of start query parameter.
+                 */
+                const newStart = response.headers.get('X-TEXT-SIZE');
+                setTimeout(exports.fetchLogsReFetch(url, newStart, onSuccess, onError), 1000);
```

Comment:
@scherler @kzantow yes this is a good point, it must only happen if they are on that screen. Obviously we can't tell if there eyes are on the screen (although there are apis for web cams, I am sure) ;)